src/SFA.DAS.AssessorService.Web/Views/Standard/OptOutStandardVersion.cshtml

```diff
@@ -58,6 +59,8 @@
                 <input asp-for="Version" value="@Model.Version" type="hidden" />
                 <input asp-for="EffectiveFrom" value="@Model.EffectiveFrom" type="hidden" />
                 <input asp-for="EffectiveTo" value="@Model.EffectiveTo" type="hidden" />
+                <input asp-for="ApprovedVersions" value="@Model.ApprovedVersions" type="hidden" />
```

Comment:
What is this for - and isn't this an enumerable anyway - you can't have a hidden for that

Reply:
Removed